app/views/miq_ae_tools/_import_export.html.haml

```diff
@@ -67,7 +67,7 @@
 
   %hr
   %h3
-    = (t = _('Reset all Datastore custom classes and instances to default'))
+    = (t = _('Reset all components in the following domains: ' + MiqAeDatastore.domain_names.join(', ')))
```

Comment:
There's an expression inside gettext, that's not gonna fly with gettext. Correctly, it should be:

```ruby
_("Reset all components in the following domains: %{domains}") % {:domains => MiqAeDatastore.domain_names.join(', ')}
```